recipes/vtk-cdat/meta.yaml

```diff
@@ -0,0 +1,87 @@
+    url: https://github.com/CDAT/VTK/archive/v{{ version }}.tar.gz
+    sha256: 48f56565b218805880078e6277ce830e2525e90a0ac0935c1ebcf577aa5ea413
+build:
+    number: 0
+    skip: true  # [win or py36]
+
+    features:
+      - mesalib  # [vtk_with_osmesa]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }} # clang 4.0.1, gcc 7.3.0
+    - cmake
+    - ninja
+
+    - {{ cdt("mesa-libgl-devel") }}  # [linux and not vtk_with_osmesa]
+    - {{ cdt("mesa-libgl") }}  # [linux and not vtk_with_osmesa]
+    - {{ cdt("mesa-dri-drivers") }}  # [linux and not vtk_with_osmesa]
+    - {{ cdt("xorg-x11-proto-devel") }}  # [linux and not vtk_with_osmesa]
+    - {{ cdt("xorg-x11-util-macros") }}  # [linux and not vtk_with_osmesa]
+    - {{ cdt("xorg-x11-server-common") }}  # [linux and not vtk_with_osmesa]
+    - {{ cdt("libXdamage") }}  # [linux and not vtk_with_osmesa]    
+    - {{ cdt("libxext-devel") }}  # [linux and not vtk_with_osmesa]
+    - {{ cdt("libxext") }}  # [linux and not vtk_with_osmesa]
+    - {{ cdt("libXfixes") }}  # [linux and not vtk_with_osmesa]    
+    - {{ cdt("libx11") }}  # [linux and not vtk_with_osmesa]
+    - {{ cdt("libx11-common") }}  # [linux and not vtk_with_osmesa]
+    - {{ cdt("libx11-devel") }}  # [linux and not vtk_with_osmesa]
+    - {{ cdt("libxt") }}  # [linux and not vtk_with_osmesa]
+    - {{ cdt("libxt-devel") }}  # [linux and not vtk_with_osmesa]
+    - {{ cdt("libxtst") }}  # [linux and not vtk_with_osmesa]
+    - {{ cdt("libxtst-devel") }}  # [linux and not vtk_with_osmesa]
+    - {{ cdt("libXxf86vm") }}  # [linux and not vtk_with_osmesa]    
+    - {{ cdt("libsm") }}  # [linux and not vtk_with_osmesa]
+    - {{ cdt("libsm-devel") }}  # [linux and not vtk_with_osmesa]
+    - {{ cdt("libice") }}  # [linux and not vtk_with_osmesa]
+    - {{ cdt("libice-devel") }}  # [linux and not vtk_with_osmesa]
+    - {{ cdt("libuuid") }}  # [linux and not vtk_with_osmesa]
+    - {{ cdt("libxcb") }}  # [linux and not vtk_with_osmesa]
+    - {{ cdt("libxau") }}  # [linux and not vtk_with_osmesa]
+    - {{ cdt("libselinux") }}  # [linux and not vtk_with_osmesa]    
+
+  host:
+    - python
+    - numpy
+    - ffmpeg
+    - x264
+    - mesalib  # [vtk_with_osmesa]
+
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - future
+    - ffmpeg
+    - x264
+    - mesalib  # [vtk_with_osmesa]
+
+test:
+    imports:
+        - vtk
+
+about:
+    home: https://github.com/cdat/vtk-cdat
+    license: BSD 3-Clause
+    license_file: Copyright.txt
+    summary:
+        vtk-cdat is vtk with modules needed by to generate CDAT's vcs plots.
+        The Visualization Toolkit (VTK) is an open-source, freely available software
+        system for 3D computer graphics, modeling, image processing, volume
+        rendering, scientific visualization, and information visualization.
+
+extra:
+    recipe-maintainers:
+        - doutriaux1
+        - sankhesh
+        - danlipsa
+        - muryanto1
+        - xylar
+        - downiec
```

Comment:
Are you willing to be maintainers?

- @doutriaux1
- @sankhesh
- @danlipsa
- @xylar
- @downiec

Reply:
I won't be working on this project for the time being, so I won't have time to maintain this.

Reply:
yes, I'm happy to be a maintainer.

Reply:
Yes, I am willing to be a maintainer as well.

Reply:
@jasonb5 are you ok to be one of the maintainers?

Reply:
Yes, I'm willing to be a maintainer as well.

Reply:
Yes I can be a maintainer.

Reply:
@scopatz all maintainers are okay to be maintainers.